readme.md

```diff
@@ -10,40 +9,43 @@
             - [Running the Container](#running-the-container)
             - [docker-compose](#docker-compose)
             - [Notes for Running Using docker-compose](#notes-for-running-using-docker-compose)
+        - [FreeNAS / FreeBSD build and run](#FreeNAS-FreeBSD)
     - [Usage](#usage)
     - [Configuration](#configuration)
 
 <!-- markdown-toc end -->
 # MovieNight stream server
-
 [![Build status](https://api.travis-ci.org/zorchenhimer/MovieNight.svg?branch=master)](https://travis-ci.org/zorchenhimer/MovieNight)
 
 This is a single-instance streaming server with chat.  Originally written to
 replace Rabbit as the platform for watching movies with a group of people
 online.
 
 ## Build requirements
-
 - Go 1.13 or newer
 - GNU Make
 
 ### Older Go Versions
-
 You can install a newer version of Go alongside your OS's distribution by
-following the guide here: [https://golang.org/doc/manage-install](https://golang.org/doc/manage-install)
+following the guide here: [https://golang.org/doc/install#extra_versions](https://golang.org/doc/install#extra_versions)
 
 Once you have that setup add an enviromnent variable named `GO_VERSION` and
 set it to the version you installed (eg, `1.14.1`).  The Makefile will now use
 the newer version.
 
 ### Compile and install
-
-To just download and run:
-
+You have to : 
+- download **git clone https://github.com/zorchenhimer/MovieNight**, go into the source directory **cd MovieNight**;
```

Comment:
Use inline code blocks for the commands.  ie, 
```
`git clone https://github.com/zorchenhimer/MovieNight`
```
instead of
```
**git clone https://github.com/zorchenhimer/MovieNight**
```

Same thing for the rest of the inline commands (the example block is fine as-is).

Reply:
Done